src/main/webapp/app/shared/util/core-drive-annotation-submission/core-drive-annotation-submission.ts

```diff
@@ -93,12 +89,8 @@ export function getVUSData(vus: Vus[]) {
   return vusDataArray;
 }
 
-function shouldExclude(onlyReviewedContent: boolean, reviewObj: Review | undefined) {
-  return (
-    reviewObj &&
-    ((onlyReviewedContent && reviewObj.added === true && reviewObj.promotedToMutation === true && reviewObj.initialUpdate === true) ||
-      (!onlyReviewedContent && reviewObj.removed === true))
-  );
+function shouldExclude(reviewObj: Review | undefined) {
+  return reviewObj && reviewObj.added === true && reviewObj.promotedToMutation === true && reviewObj.initialUpdate === true;
```

Comment:
Based on the comment in firebase.model.ts, `added` and `promotedToMutation` can coexist. And look at firebase-gene-service, when promotedToMutation is true, added is always true. Could you take a look?

Reply:
Yes this appears to be the case. I also couldn't find where `intialUpdate` is ever `true`. Does that mean this should exclude logic will always be `false`?

Reply:
`return reviewObj && (reviewObj.added === true || reviewObj.promotedToMutation === true || reviewObj.initialUpdate === true);` based on the conversation, we should update to this.
Could you also update the comment to reflect what Calvin said?